resource/storage.go

```diff
@@ -155,7 +155,7 @@ func (s storageWrapper) MultiGet(ctx context.Context, ids []interface{}) (items
 }
 
 // Find tries to use storer MultiGet with some pattern or Find otherwise
-func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage int) (list *ItemList, err error) {
+func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage, offset int) (list *ItemList, err error) {
```

Comment:
> I hope I could explain it.

Maybe we are talking past each other than, and maybe I replied a bit quickly to your previous comment. I do agree on the semantic differences for the REST API point of view.

However, would you agree that these semantic differences are irrelevant for the Storage inteface / storage backends, which is what I am suggesting to simplify here?
>